cli/src/cmd/forge/build/core.rs

```diff
@@ -153,7 +153,7 @@ impl<'a> From<&'a CoreBuildArgs> for Figment {
         remappings
             .extend(figment.extract_inner::<Vec<Remapping>>("remappings").unwrap_or_default());
         remappings.sort_by(|a, b| a.name.cmp(&b.name));
```

Comment:
we also need to sort exactly like:

https://github.com/foundry-rs/foundry/blob/1058c8afb2f5b682d4f40b836c55006734e06945/config/src/providers/remappings.rs#L129-L130